### ISSUE TITLE
Stremio 4.0 alpha 21

### ISSUE THREAD

**Issue by Harith163** [Stremio/stremio-bugs#91]
Hi!
I know that it is still in alpha, but It is extremely unstable. If I tried to switch from board to library, it crashed. Maximising crashed it. You probably are already aware of this but just thought that I would chip in anyway.

GTX 970m
i7-6700HQ 
all latest drivers.

Sorry for the disturbance!

**Comment by Ivshti** [comment]
Hello,

Thanks for reporting this. We have not seen this behaviour, otherwise we wouldn't even be uploading it. 

Does it just quit? Does it crash consistently?

Which windows versioN?

**Comment by Harith163** [comment]
Windows 10 Version 1607 Build 14393.693
It doesn't just quit. Only on interacting, It crashed. I launched the application 5 times and it crashed everytime, but doing different actions. I couldn't notice any pattern. Since it only happened everytime things changed, I assumed it was some kind of graphical issue. That is why I listed the gpu and cpu.

**Comment by Harith163** [comment]
I checked out the online alpha and it seems to be pretty stable. It however failed to load any episode and gave the connection timeout error message. The same episode worked fine on the same add-on on version 3.6.5 mere minutes earlier.

P.S: I really look forward to 4.0 It looks and sounds awesome! I can't wait!

**Comment by Harith163** [comment]
Also, is it possible to have the remote as an app? That would be real cool.

**Comment by Ivshti** [comment]
@Harith163 we'll add remote to our android app (and PS iOS app coming out in February)

Can you please make this test:

Start the 4.0 alpha, don't touch it

Go to the web alpha (app.strem.io) and see if you can play things from there

**Comment by Harith163** [comment]
Should I just quit 3.6.5 or completely uninstall it?

**Comment by Ivshti** [comment]
quit works, but there's a tiny (theoretically none) chance that 3.6.5 is interfering with 4.0

**Comment by Harith163** [comment]
I did a full uninstall( even went into task manager and removed lingering stuff). And installed 4.0.21 and opened it. The web alpha still doesn't load though. 
The desktop shell seems slightly more stable but still crashes on the regular and also doesn't load anything.
Proof:
![screenshot 4](https://cloud.githubusercontent.com/assets/18382195/22318171/7e38bfc2-e346-11e6-931d-0c3c3337ff6a.png)
![screenshot 5](https://cloud.githubusercontent.com/assets/18382195/22318173/7e39ab4e-e346-11e6-8a13-4d49f181389b.png)
![screenshot 6](https://cloud.githubusercontent.com/assets/18382195/22318172/7e38c92c-e346-11e6-9098-c999dc43d04e.png)

**Comment by Ivshti** [comment]
@Harith163 
can you try a few variations of DirectX configuration

Install DirectX 9 / Uninstall DirectX9
Install DirectX 11 / Uninstall DirectX 11

**Comment by Harith163** [comment]
Hey! Sorry for the late update.

DirectX doesn't seem to be the issue here. I tried both 9 and 11. I didn't try 12 because you didn't seem to mention it.
I tried the latest stremio alpha that was uploaded to reddit. Its far more stable and it crashed only once. It still doesn't load any videos though. The first time I launched the app, I got this message-
![screenshot 7](https://cloud.githubusercontent.com/assets/18382195/22723610/df2bd016-ed8d-11e6-88a8-53b4044db4c1.png)

After that the app hung and crashed.

I then promptly reloaded the app and things went smoothly(barring a few graphical stutters; which I honestly don't mind.). I tried loading the same episode as mentioned in previous posts. The videos never went past the loading screen though.
![screenshot 8](https://cloud.githubusercontent.com/assets/18382195/22723647/2a40bf80-ed8e-11e6-9727-4702e17c3982.png)
